### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

fixes https://github.com/kubeflow/kubeflow/issues/4025

@jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1254)

### CODE REVIEW

**Comment by nrchakradhar** [comment]
LGTM

File: content/docs/other-guides/virtual-dev/getting-started-minikube.md [hunk 1]
```diff
@@ -93,15 +93,15 @@ chmod +x setup-minikube.sh
 **KUBEFLOW_TAG** is a tag corresponding to the version to checkout such as {{% kf-latest-version %}} or `master`.
 
 The script asks for some config input as shown below:
-![ConfigInputs](../ConfigInputs.png)
+![ConfigInputs](../../started/ConfigInputs.png)
```
Comment (sarahmaddox) [inline content/docs/other-guides/virtual-dev/getting-started-minikube.md]: The images are still broken - see preview:
https://deploy-preview-1254--competent-brattain-de2d6d.netlify.com/docs/other-guides/virtual-dev/getting-started-minikube/

For the path to the image, it's best to use a relative path that starts with `/docs/...`. While you're here, please would you add styling as well, to make the images look better. The pattern looks like this:

```
<img src="/docs/images/kubeflow-deployment.png" 
  alt="Kubeflow deployment UI"
  class="mt-3 mb-3 border border-info rounded">
```

The above example comes from this file: https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/deploy-ui.md

Comment (yeya24) [reply content/docs/other-guides/virtual-dev/getting-started-minikube.md]: Thanks for your suggestions!

**Comment by yeya24** [comment]
> /assign
> /ok-to-test

See: https://deploy-preview-1254--competent-brattain-de2d6d.netlify.com/docs/other-guides/virtual-dev/getting-started-minikube/ 
I believe it is good now.

**Comment by sarahmaddox** [comment]
Great, thanks @yeya24!

/lgtm
/approve

**Pull request merged**